### ISSUE TITLE
multitap

### ISSUE THREAD

**Issue by DarkKinof** [libretro/bsnes2014#23]
Hi, a first, thanks for your great work with retroarch/libretro.
This report is for SNES MULTITAP support.
I found closed issue about this problem, and check another time libretro cores (v1.3.6)

Today SNES9x core work without any problem with multitap games.
But Bsnes w/o mercury don´t work, games show multitap detection (bomberman, Tiny Toon Adventures - Wacky Sports Challenge, etc) like SNES9x core, but 3, 4 and 5 players don´t move.

I user same cfg file in both cases, only change core DLL into configuration file.
Regards.

**Comment by newoski** [comment]
Confirmed. Same problem.

**Comment by Cobra84** [comment]
As far as I know, it has never worked.

**Comment by hizzlekizzle** [comment]
Does it work with mercury?

**Comment by newoski** [comment]
No. The only core I got it to work with, to some degree, was the C++86 one. I can't recall the name off the top of my head, but it only seems to work for select games -- Looney Tunes B-Ball is the game I got it working for. Others did not work

**Comment by DarkKinof** [comment]
I people, please, how can i help to solve this problem?
Could i do some test or another actions and upload reports?
Regards.
